libs/gtkmm2ext/po/fr.po

```diff
@@ -31,27 +32,27 @@ msgstr "Désactivé"
 
 #: action_model.cc:83
 msgid "Main_menu"
-msgstr ""
+msgstr "Menu principal"
 
 #: action_model.cc:85
 msgid "JACK"
 msgstr ""
 
 #: action_model.cc:87
 msgid "redirectmenu"
-msgstr ""
+msgstr "Menu redirection"
 
 #: action_model.cc:89
 msgid "Editor_menus"
-msgstr ""
+msgstr "Menus éditeur"
 
 #: action_model.cc:91
 msgid "RegionList"
-msgstr ""
+msgstr "Liste des régions"
 
 #: action_model.cc:93
 msgid "ProcessorMenu"
-msgstr ""
+msgstr "Menu processeur"
```

Comment:
Pas certains que ces choses-là soient à traduire. Où sont-elles affichées pour l'utilisateur ?
Not sure if these things need to be translated. Where are they displayed to the user?